src/types.ts

```diff
@@ -81,4 +83,7 @@ export interface PostResults {
 export interface PutResults {
   status: string
   resultURL: URL
-}
\ No newline at end of file
+}
+
+export type XcodeCoverageFileReport = NodeJS.Dict<string>
+export type XcodeCoverageReport = NodeJS.Dict<XcodeCoverageFileReport>
```

Comment:
```suggestion
export type XcodeCoverageFileReport = Record<string, string>
export type XcodeCoverageReport = Record<string, XcodeCoverageFileReport>
```

Reply:
`UploaderEnvs` should also be updated according to the suggested changes in a separate PR.

Reply:
@mitchell-codecov sometimes it resolves to `undefined` I think